src/calculateRentalCost.js

```diff
@@ -29,6 +29,13 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days <= 2) {
+    return (days * 40);
+  } else if (days < 7) {
+    return ((days * 40) - 20);
+  } else if (days >= 7) {
+    return ((days * 40) - 50);
```

Comment:
move "magic" numbers into variables. It could be explicit, what your numbers mean